docs/content/docs/build/containers.md

```diff
@@ -64,13 +64,20 @@ const { container, services } =
 const containerId = await container.attach();
 ```
 
-### Attaching a container
+Notes:
+
+- `client` represents an object defined by the service-specific client library. See the documentation for the service you are using for more details about how to use its service-specific client library.
+- It is a good practice to deconstruct the object that is returned by `createContainer` into its two main parts; `container` and `services`. For an example of the use of the latter, see [Working with the audience]({{< relref "audience.md#working-with-the-audience" >}}).
 
-A newly created container is in a *detached* state. This is the point where you can create initial data to populate your
-shared objects if needed. A detached container is not connected to the Fluid service and no data is shared with other clients.
+A newly created container is in a *detached* state. A detached container is not connected to the Fluid service and no data is shared with other clients.
```

Comment:
I tried to separate **attached** and **connected** terminology. This line also needs a change.